active/RadioButtonsAPI.md

```diff
@@ -0,0 +1,105 @@
+# Background
+
+RadioButtons is our answer to enabling better keyboarding and narrator support for RadioButtons that need to be presented in a related list.
+
+Currently, the only way to group RadioButtons is through using a StackPanel and individually labeling each RadioButton through its GroupName property in order for narrator to read out the options somewhat understandably.
+
+However, using that method still left gaps in the narrator experience and StackPanel does not behave the way a user would expect when navigating the list via keyboard - RadioButtons was introduced to help solve those issues.
+
+# Description
+
+RadioButtons is a class extension of ListView that enables you to create related lists of RadioButtons easily, while also fully supporting keyboarding and narrator functionality.
```

Comment:
It contains a ListView but it intentionally does not derive from ListView to avoid confusing the API with all the ListView API. This may or may not be the right choice but we shouldn't say it's a "class extension".

Reply:
Okay, I'll just say "control" then.